modules/tinymce/src/plugins/table/main/ts/actions/ResizeHandler.ts

```diff
@@ -71,6 +64,17 @@ export const getResizeHandler = function (editor: Editor): ResizeHandler {
       // For preserve table we want to always resize the entire table. So pretend the last column is being resized
       const col = Settings.isPreserveTableColumnResizing(editor) || isRightEdgeResize ? getNumColumns(table) - 1 : 0;
       Adjustments.adjustWidth(table, width - startW, col, resizing, tableSize);
+    // Handle the edge case where someone might fire this event without resizing.
+    // If so then we need to ensure the table is still using percent
+    } else if (Util.isPercentage(startRawW)) {
```

Comment:
I was talking to @ltrouton about this and we were wondering if we should just remove this block? As core will never fire the resize event if the sizes hasn't changed, so this is a very large edge case.

What do others think?

Reply:
I'm like a 4/5 for agreeing that you should remove it, but I'm only 1/5 confident.

Reply:
Which event is this referring to? Just wondering how likely it is that an integrator will fire it....

Reply:
The `ObjectResized` event, specifically for tables.

Reply:
...I'm leaning towards leaving it in then, just in case. We know some people hook into that event, so even though it's an edge case, since we already have code for it it seems worth dealing with?

Reply:
I'll leave it as is then.